src/itest/resources/features/company-insolvency.feature

```diff
@@ -23,3 +50,11 @@ Feature: Process company insolvency information
     Examples:
       | companyNumber | result                     |
       | CH3634545     | retrieve_by_company_number |
+
+  Scenario: Processing company insolvency information while database is down
```

Comment:
Covers the scenario:
<img width="762" alt="image" src="https://user-images.githubusercontent.com/16381559/164451548-45704505-af89-4268-ada0-bf54f3ebbdee.png">